.circleci/config.yml

```diff
@@ -284,6 +284,24 @@ defaults:
             name: Install build dependencies
             command: ./.circleci/osx_install_dependencies.sh
 
+  - steps_install_python_dependencies: &steps_install_python_dependencies
+      steps:
+        - run:
+            name: Install python dependencies
+            command: |
+                sudo apt update
+                sudo apt install python3-pip --assume-yes --no-install-recommends
+                python3 -m pip install requests --user
+
+  - steps_install_foundry: &steps_install_foundry
+      steps:
+        - run:
+            name: Install foundry
+            command: |
+                # Note: Currently Foundry only has nightly builds and there is no release checksum
+                wget https://github.com/foundry-rs/foundry/releases/download/nightly/foundry_nightly_linux_amd64.tar.gz --output-document /tmp/foundry.tar.gz
```

Comment:
What's `releases/nightly` actually? Does not look like a tag (that would be `tree/nightly`) but seems to be pointing at an old tag anyway. It's from August 2022 but I see there are fresh nightlies available even from a few hours ago.

Is this specifically a revision recommended by them for stable use or just some old, forgotten tag?

Reply:
It points to the file `foundry_nightly_linux_amd64.tar.gz` in the nightly release page: https://github.com/foundry-rs/foundry/releases/tag/nightly which was supposed to be the latest. But I fixed it now to consider the commit hash.